src/Generators/Generator.php

```diff
@@ -294,15 +290,19 @@ public function save($files, $answers, &$results)
 
     /**
      * @return string the root path of the template files that are currently being used.
-     * @throws InvalidConfigException if [[template]] is invalid
+     * @throws ReflectionException
      */
-    public function getTemplatePath()
+    public function getTemplatePath(): string
     {
+        if ($this->template === 'default') {
+            return $this->defaultTemplate();
+        }
+
         if (isset($this->templates[$this->template])) {
             return $this->templates[$this->template];
         }
 
-        throw new InvalidConfigException("Unknown template: {$this->template}");
+        throw new RuntimeException("Unknown template: {$this->template}");
```

Comment:
You should create and use `InvalidConfigException`. It will be called in other places too